sgemm.cpp

```diff
@@ -586,15 +583,15 @@ class tinyBLAS_Q0_ARM {
 };
 #endif // __ARM_FEATURE_DOTPROD
 
-#if defined(__AVX2__) || defined(__AVX512F__)
+#if defined(__AVX2__) || defined(__AVX512F__) || defined(__AVX__)
 template <typename TA, typename TB, typename TC>
-class tinyBLAS_Q0_AVX2 {
```

Comment:
This should be renamed `tinyBLAS_Q0_X86`

Reply:
See my other comment, the current code is AVX only.